README.md

```diff
@@ -403,8 +403,8 @@ project:
 Regal will automatically search for a configuration file (`.regal/config.yaml`
 or `.regal.yaml`) in the current directory, and if not found, traverse the
 parent directories either until either one is found, or the top of the directory
-hierarchy is reached. If no configuration file is found, Regal will use the
-default configuration.
+hierarchy is reached. If no configuration file is found, and no file is found at
+`~/.config/regal/config.yaml` either, Regal will use the default configuration.
```

Comment:
This is good, but I'd probably add a section on its own about "User-level configuration" or something. Where we also explain when we think it's appropriate to use one, and that regular projects should rather check in their Regal config along with their policies.